docs/tutorials/intro.md

```diff
@@ -0,0 +1,177 @@
+4. The GIS object properties, annotations and filters of a selected GIS layer.
+5. The Jupyter toolbar menu which you will use with Jupyter Notebooks.
+6. The Log console (which you can you for debugging).
+7. Right Sidebar showing the property inspector (active in notebooks), the debugger, and GIS object properties, annotation and filters.
+
+## Adding your first layers
+
+We will start the JupyterGIS application and create a basic map.
+
+**The goal for this lesson: To get started with an example map.**
+
+:::{admonition} Get JupyterGIS
+:class: tip
+Before beginning this exercise, JupyterGIS must be installed on your computer (see [Installation instructions](https://geojupyter.github.io/jupytergis/install.html)) or you can use an online version of JupyterGIS (such as [![Jupyterlite badge](https://jupyterlite.rtfd.io/en/latest/_static/badge.svg)](https://geojupyter.github.io/jupytergis/lite/lab/index.html?path=france_hiking.jGIS/)).
+:::
+
+Launch JupyterGIS from your terminal or start an online version of JupyterGIS depending on what you decided to use for this exercise.
+
+```bash
+jupyter lab
+```
+
+### Add a vector Layer
+
+Follow along to add a vector layer available online.
+
+1. Click on the GIS File Icon in the "Other" section of the Launcher to create a new JGIS Editor. You will have a new, blank map.
+   ![JGIS Editor](../assets/images/JGIS_editor.png)
+2. Open Layer Browser (see image below) and select for instance _OpenStreetMap.Mapnik_. An interactive map will appear and you will be able to zoom in and zoom out with your mouse.
+   ![Open Layer Browser](../assets/images/OpenLayerBrowser.png)
+3. Click on the **+** to add another type of layer and select "New Shapefile Layer" to add a new vector layer (stored as a shapefile).
+   ![Add a Vector Layer](../assets/images/addVectorLayer.png)
+4. Set the path of the new shapefile Layer to `https://www2.census.gov/geo/tiger/GENZ2015/shp/cb_2015_us_county_500k.zip` and click **Ok**.
+   ![Set Shapefile Layer](../assets/images/CreateShapefileLayer.png)
+5. You should then be able to see the new Shapefile Layer on the map and when clicking on the **Custom Shapefile Layer** (see figure below), you should also see the details of the object properties on the right of the map. The map being interactive, you can zoom in and out as you wish to focus on your area of interest (here over the US).
+   ![View Shapefile Layer](../assets/images/viewShapefileLayer.png)
+
+:::{admonition} Why don't we add a local shapefile?
+:class: tip
+One of the main strength of JupyterGIS is its collaborative capabilities. To be able to collaborate, it is much easier to work with data shared and available online.
+:::
+
+```{exercise-start}
+:label: add-vector-layer
+```
+
+- Add a new Shapefile Layer to your map:
+
+```
+https://docs.geoserver.org/stable/en/user/_downloads/30e405b790e068c43354367cb08e71bc/nyc_roads.zip
+```
```

Comment:
Both layers display as "Custom Shapefile layer" at this point in the tutorial, and same for the sources. It would be nice to give them names early in this tutorial to reduce the cognitive load of the learner! Then we can demonstrate renaming later in the tutorial without having to pay the cognitive load price.